services/QuillLMS/config/routes.rb

```diff
@@ -385,7 +386,12 @@
           get 'level_zero_concepts_with_lineage'
         end
       end
-      resources :users,                   only: [:index]
+      
+      resources :users, only: [:index] do
+        resources :app_settings, only: [:index] 
+        get 'app_setting(/:name)' => 'app_settings#show'
```

Comment:
This line cannot be completely "rails-y" since it is not a RESTful route; we want the parameter to be name rather than id, so that the client does not need to care about AppSetting ids.

Reply:
You could route this the rails-y way, but the param in the controller would be `:id` instead of `:name`.  That might be confusing enough that it's worth making the router a bit more complicated to name the param the right thing, so I'm okay with this.

Reply:
You can change the name of the `param` for a resource: https://guides.rubyonrails.org/routing.html#overriding-named-route-parameters

This seems like a reasonable setup since `name` will likely be used in place of `id` for any route we add.

```ruby
resources :app_settings, only: [:index, :show], param: :name